.github/workflows/ghcr_app.yml

```diff
@@ -79,12 +78,12 @@ jobs:
           path: /tmp/${{ matrix.image }}_${{ steps.capture-last-tag.outputs.last_tag }}_${{ matrix.platform }}.tar
           retention-days: 14
 
-  # Push the OpenHands and sandbox Docker images to the ghcr.io repository
+  # Push the OpenHands Docker images to the ghcr.io repository
   ghcr_push:
     name: Push App Image
     runs-on: ubuntu-latest
     needs: [ghcr_build]
-    if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/') || (github.event_name == 'pull_request' && github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main')
+    if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')
```

Comment:
Is this intended?

Reply:
Yes. It shouldn't be needed. The main commits should run with github.ref == 'refs/heads/main'.

If my PR completes and this runs, it will solidify my suspicion that it was the concurrency all along.